composites/enforce-pmc-scripts/enforce-pmc-scripts.sh

```diff
@@ -5,33 +5,34 @@ HAS_ERROR=""
 NEED_UPDATE_PACKAGE_JSONS=()
 NEW_JS_FILES=()
 
+if [[ -f .pmc-scripts ]]; then
+  # load the env JS_IGNORE_PATTERNS from the .pmc-scripts file
+  export $(grep -v '^#' .pmc-scripts  | xargs)
```

Comment:
Shellcheck strikes again 😆 

```suggestion
  export "$(grep -v '^#' .pmc-scripts  | xargs)"
```

Reply:
Ah, in that case, we can add `# shellcheck disable=SC2046` to silence the warning and ensure no one quotes it in the future.

```suggestion
  # shellcheck disable=SC2046
  export $(grep -v '^#' .pmc-scripts  | xargs)
```